CONTRIBUTING.md

```diff
@@ -78,6 +79,56 @@ branches that minor and patch releases will be tagged. In some cases, it may
 be necessary to open PRs for bugfixes directly against stable branches, but
 this should generally not be the case.
 
+### Support and guarantees
+
+Baremetal-operator maintains the most recent release/releases for all supported
+APIs and contract versions. Support for this section refers to the ability to
+backport and release patch versions;
+[backport policy](#backporting) is defined above.
+
+- The API version is determined from the GroupVersion defined in the top-level
+  `apis/` package.
+
+- The EOL date of each API Version is determined from the last release available
+  once a new API version is published.
+
+| API Version  | Supported Until                                               |
+| ------------ | ------------------------------------------------------------- |
+| **v1alpha1** | TBD (current latest)                                          |
+| **v1beta1**  | Upcoming (Proposal [PR](https://github.com/metal3-io/metal3-docs/pull/332))                                         |
+
+- For the current stable API version (v1alpha1) we support the two most recent
+  minor releases; older minor releases are immediately unsupported when a new
+  major/minor release is available.
+
+- Once we have v1beta1 API, we will determine support policies for older API
```

Comment:
@dtantsur this is a placeholder for the policy to support older APIs once we have v1beta1 API.